start.sh

```diff
@@ -11,4 +11,5 @@ cat > /etc/rabbitmq/rabbitmq.config <<EOF
 	{rabbit, [{default_user, <<"$1">>},{default_pass, <<"$2">>},{tcp_listeners, [{"0.0.0.0", 5672}]}]}
 ].
 EOF
+chown rabbitmq:rabbitmq /var/lib/rabbitmq/mnesia
```

Comment:
Shouldn't this be `/var/reana/rabbitmq/mnesia` ?

Reply:
Nope, inside the container the path is well `/var/lib/...` mapped from the host's `/var/reana/...`.